scripts/test

```diff
@@ -54,9 +54,10 @@ compose() {
   (
     cd "${projectRoot}"
     docker-compose \
-    -f docker-compose.yaml \
-    -f docker-compose.test.yaml \
-    $@
+      -f docker-compose.yaml \
+      -f docker-compose.test.yaml \
+      -p ffc-demo-web-test \
```

Comment:
The hard coded project name could be substituted for the `containerName` with `-test` appended

Reply:
Good spot. I've used a new variable `${projectName}` to reflect what it's being used for.